lib/index.js

```diff
@@ -0,0 +1,52 @@
+Object.defineProperty(exports, "__esModule", {
```

Comment:
hmm... I think so? the `main` key in our package.json have pointed when people use 
`import {renderFunction} from 'hypernova-function'`, it will find hypernova-function the package.json first to check the `main` key, to find the entry file the module will be load.

So, this compile just change the module bundle rules from ES6 module to node CommonJS module rules, to support when the client doesn't have compiler to compile `import` to `require`